hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1151,7 +1151,7 @@
       uvm_test_seq: chip_sw_all_escalation_resets_vseq
       sw_images: ["//sw/device/tests:alert_handler_lpg_sleep_mode_alerts_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+en_scb=0", "+sw_test_timeout_ns=3000_000_000", "+bypass_alert_ready_to_end_check=1", "+avoid_inject_fatal_error_for_ips=sram_ctrl_main,flash_ctrl,lc_ctrl"]
+      run_opts: ["+en_scb=0", "+sw_test_timeout_ns=3000_000_000", "+bypass_alert_ready_to_end_check=1", "+avoid_inject_fatal_error_for_ips=sram_ctrl_main,flash_ctrl,lc_ctrl,otp_ctrl"]
```

Comment:
This avoids all errors for these IPs. I wonder if that excludes too many, meaning if the fatal alerts in these IPs are a superset of those listed in https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/tests/sim_dv/all_escalation_resets_test.c;drc=539c4bb6e46c2e36e885e63b0820e7eaf3399ed3;l=1186.

Reply:
@matutem chatted with me about this offline, pointing out that it's specific alert causes that lead to the stoppage of software's forward progress.

We have decided to go ahead and omit otp_ctrl's fatal alerts from this test, since precisely tracking the answer to the question, "Can we process otp_ctrl's fatal alerts after only getting an NMI (without a reset)?" is not high-priority.